src/Parser/Mage/Fire/Modules/Features/DeathTracker.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import CoreDeathTracker from 'Parser/Core/Modules/DeathTracker';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+
+class DeathTracker extends CoreDeathTracker {
```

Comment:
I think this suggestion should be a part of the core version of this module so that it's implemented for everyone for free.

How about making the `You can also use defensive spells like <SpellLink id={SPELLS.BLAZING_BARRIER.id}/> or <SpellLink id={SPELLS.ICE_BLOCK.id}/> to avoid taking unavoidable damage. Additionally, ensure you are using Health Potions and Healthstones to heal yourself when you are very low.` part a property in the core so that we can use the config setting in CombatLogParser to change it as desired:

```jsx
deathTracker: [DeathTracker, { extraSuggestion: <Wrapper>You can also use defensive spells like <SpellLink id={SPELLS.BLAZING_BARRIER.id}/> or <SpellLink id={SPELLS.ICE_BLOCK.id}/> to avoid taking unavoidable damage. Additionally, ensure you are using Health Potions and Healthstones to heal yourself when you are very low.</Wrapper> }],
```

Keeping it that simple wouldn't require everyone to extend and override the core module.

Reply:
I agree, all the DeathTracking stuff should be core, only different between specs that I can think of would be the text of the suggestion.